protocol/storage/client_test.go

```diff
@@ -208,6 +211,10 @@ func newTestClientNode(responder func(request interface{}) (interface{}, error))
 	}
 }
 
+func (tcn *testClientNode) SignBytes(data []byte, addr address.Address) (types.Signature, error) {
+	return nil, nil
```

Comment:
Blocking: set some value for the signature and assert that the signed proposal carries it.